app/mailers/publisher_mailer.rb

```diff
@@ -71,11 +71,10 @@ def verify_email(publisher)
 
     if @publisher.pending_email.blank?
       begin
-        raise "SMTP To address must not be blank for PublisherMailer#verify_email"
+        raise "SMTP To address must not be blank for PublisherMailer#verify_email for publisher #{@publisher.id}"
       rescue => e
         require 'sentry-raven'
-        Raven.capture_exception(e,
-                                publisher: @publisher)
+        Raven.capture_exception(e)
       end
     end
     mail(
```

Comment:
Won't mail still be called after the exception is caught?

Reply:
Yes, should we eliminate the `mail` call in this case?

Reply:
I'd say yes. Throw it in an `else` block, and maybe swap the `if` logic so the error condition is the `else`.